Makefile

```diff
@@ -1,6 +1,6 @@
 NODE_DIR?=/opt/compiler-explorer/node
-NPM:= $(shell env PATH="$(NODE_DIR)/bin:$$PATH" which npm)
-NODE:= $(shell env PATH="$(NODE_DIR)/bin:$$PATH" which node || env PATH="$(NODE_DIR)/bin:$$PATH" which nodejs)
+NPM:=$(shell env PATH=$(NODE_DIR)/bin:$(PATH) which yarn)
```

Comment:
Is the change from $$ to $( right? If I recall correctly, we changed to $$ as per @apardoe's suggestion

Reply:
Ah, I just copied and pasted from one of the merges. Will fix that up.